### PR TITLE
48 add hash map page

### PR DESCRIPTION
**Pull request by arch92**
Added Hash Map Page.

### CODE REVIEW

**Comment by arch92** [comment]
@elaguerta Can you review this?

File: topics/Hashmaps.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+[First Unique Character in a String](https://leetcode.com/problems/first-unique-character-in-a-string/) | [leetcode's endorsed solution](https://leetcode.com/problems/first-unique-character-in-a-string/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/linked-lists)
+[Intersection of Two Arrays II](https://leetcode.com/problems/intersection-of-two-arrays-ii/) | [leetcode's endorsed solution](https://leetcode.com/problems/intersection-of-two-arrays-ii/solution/) | please contribute your solution to our [code examples!](https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/linked-lists)
+
+### Medium problems
+Problem Statement | Solution | WWC example
+--- | --- | ---
+[Longest Substring Without Repeating Characters](https://leetcode.com/problems/longest-substring-without-repeating-characters/) | [leetcode's endorsed solution](https://leetcode.com/problems/longest-substring-without-repeating-characters/solution/) | please contribute your solution to our [code examples!]()
```
Comment (elaguerta) [inline topics/Hashmaps.md:34]:
```suggestion
[Longest Substring Without Repeating Characters](https://leetcode.com/problems/longest-substring-without-repeating-characters/) | [leetcode's endorsed solution](https://leetcode.com/problems/longest-substring-without-repeating-characters/solution/) | please contribute your solution to our [code examples!]((https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/hashmaps)
```

File: topics/Hashmaps.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+[Top K Frequent Words](https://leetcode.com/problems/top-k-frequent-words/) | [leetcode's endorsed solution](https://leetcode.com/problems/top-k-frequent-words/solution/) | please contribute your solution to our [code examples!]()
```
Comment (elaguerta) [inline topics/Hashmaps.md:35]:
```suggestion
[Top K Frequent Words](https://leetcode.com/problems/top-k-frequent-words/) | [leetcode's endorsed solution](https://leetcode.com/problems/top-k-frequent-words/solution/) | please contribute your solution to our [code examples!]((https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/hashmaps)
```

File: topics/Hashmaps.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### Hard problems
+Problem Statement | Solution | WWC example
+--- | --- | ---
+[LFU Cache](https://leetcode.com/problems/lfu-cache/) | [leetcode's endorsed solution](https://leetcode.com/problems/longest-substring-without-repeating-characters/solution/) | please contribute your solution to our [code examples!]()
```
Comment (elaguerta) [inline topics/Hashmaps.md:40]:
```suggestion
[LFU Cache](https://leetcode.com/problems/lfu-cache/) | [leetcode's endorsed solution](https://leetcode.com/problems/longest-substring-without-repeating-characters/solution/) | please contribute your solution to our [code examples!]((https://github.com/WomenWhoCode/wwcsf-algos/tree/master/code-examples/hashmaps)
```

**Review by elaguerta — CHANGES_REQUESTED** [review]
Hey Archana, this is looking great! Please address the following:
1. Add a folder to `/code-examples` called `hashmaps`. This will be where people can submit code examples. 
2. I added suggestions in-line to provide correct links to the new code-examples/hashmaps  folder, please commit these. 
3. Add a link to this page to the main README, in this section:
https://github.com/WomenWhoCode/wwcsf-algos#3-resources-and-problem-sets-by-category
4. Add this problem to the Hard section: https://leetcode.com/problems/maximum-frequency-stack/
5. Change your PR from draft status to normal, so we can merge it in!

Thank you!

**Review by elaguerta — APPROVED** [review]
Hey @arch92 I completed the revisions, so you're done! Thanks so much for your contribution. I'm looking forward to sharing this with the group next week!

**Pull request merged**